supportedBackends/tes/src/main/scala/cromwell/backend/impl/tes/TesAsyncBackendJobExecutionActor.scala

```diff
@@ -380,7 +380,7 @@ class TesAsyncBackendJobExecutionActor(override val standardParams: StandardAsyn
       HttpRequest(method = HttpMethods.POST, uri = s"$tesEndpoint/${job.jobId}:cancel")
     ) onComplete {
       case Success(_) => jobLogger.info("{} Aborted {}", tag: Any, job.jobId)
-      case Failure(ex) => jobLogger.warn("{} Failed to abort {}: {}", tag, job.jobId, ex.getMessage)
+      case Failure(ex) => jobLogger.warn("{} Failed to abort {}", tag, job.jobId, ex)
```

Comment:
Although this might _look_ wrong, it's actually the format slf4j expects for including a Throwable at the end of a log builder  argument list, and note that means we don't need a special placeholder in the string